### ISSUE TITLE
Feature Request - Item particles (eat)

### ISSUE THREAD

**Issue by JustTimmNL** [sp614x/optifine#5326]
Item Particle
CIT Feature request

It would be cool if there was a feature that changed the particle of an custom CIT item when you eat it other particles like when a snowball hits the ground.

**Comment by PixelatedVolume** [comment]
I agree doing this through CIT directly would be convenient, but if you want to chance the particles right now you could try making a new model for your item so you can specify the _particles_ field.
